docs/build-on-polkadot/xcm/asset-transfer-api.md

```diff
@@ -0,0 +1,491 @@
+
+const assetsApi = new AssetTransferApi(api, specName, safeXcmVersion);
+```
+-->
+
+### Using helper function from library
+
+For a simpler setup process, you can leverage the `constructApiPromise` helper function provided by the library. 
+
+<!-- This method automatically initializes the Polkadot.js API, retrieves the specName, and determines the safe XCM version. -->
+
+```javascript
+import { AssetTransferApi, constructApiPromise } from '@substrate/asset-transfer-api';
+
+async function main() {
+    const { api, specName, safeXcmVersion } = await constructApiPromise('<INSERT_WEBSOCKET_URL>');
+
+    const assetsApi = new AssetTransferApi(api, specName, safeXcmVersion);
+
+    // Your code using assetsApi goes here
+}
+
+main();
+```
+
+!!!note
+    The code examples are enclosed in an async main function to provide the necessary asynchronous context. However, if you're already working within an async environment, you can use the code directly. The key is to ensure you're in an async context when working with these asynchronous operations, regardless of your specific setup.
+
+
+## Asset Transfer API Overview
+
+The AssetTransferApi provides a powerful method for cross-chain asset transfers: `createTransferTransaction`. This function allows you to create XCM transactions for transferring assets or native tokens between different chains.
+This method takes several parameters to specify the details of the transfer, including the destination chain, recipient address, assets to be transferred, their amounts and an optional parameter for further customization. It is able to infer what kind of transaction is necessary given the inputs. When sending cross-chain transfers, the API performs extensive validation to ensure the inputs are valid, and the assets either exist or don't.
+
+!!! note
+	The `createTransferTransaction` function is designed to be a utility that simplifies the creation of the transaction. It does not sign or submit the created transaction on the blockchain. It simply generates the transaction in the requested format (e.g., payload, call, or submittable). After obtaining the transaction from the `createTransferTransaction` function, you will need to handle the signing and submission process separately.
+
+```javascript
+public async createTransferTransaction<T extends Format>(
+  destChainId: string,
+  destAddr: string,
+  assetIds: string[],
+  amounts: string[],
+  opts: TransferArgsOpts<T> = {},
+): Promise<TxResult<T>>
+```
+
+### Parameters
+
+- `destChainId` ++"string"++ - ID of the destination chain ('0' for Relay chain, other values for parachains)
+- `destAddr` ++"string"++ - Address of the recipient account on the destination chain
+- `assetIds` ++"string[]"++ - Array of asset identifiers to be transferred
+  
+  If a token or assetId is provided as input, the API will resolve to using the `tokens` pallet. When no asset is passed in, the API will default to using the `balances` pallet.
```

Comment:
On the Asset Hub overview, if I'm not referring to something explicitly typed on js or ts, I'm using the standard `asset ID` or `asset IDs`. Maybe we should agree on which form we are gonna use throughout the website so it's consistent (again, I'm referring to those situations where we are NOT talking about the code directly)

Reply:
You are right about this. I will replace it with asset ID/s